test/release/examples/patterns/readcsv.chpl

```diff
@@ -0,0 +1,286 @@
+/* This example demonstrates how to read in a CSV file.
+
+   The example csv file is from Kaggle,
+   https://www.kaggle.com/allen-institute-for-ai/CORD-19-research-challenge?select=metadata.csv
+   The included metadata-20lines.csv only has 20 lines of data including the
+   header row that has the column names.
+
+   Each line in csv files is a row of data.  Columns are separated by commas.
+   Some of the cells of data have quotes because the data string itself includes
+   a comma or commas.
+
+   There are many options in terms of data structures that such data could be
+   read into.  Here are the two approaches illustrated here.
+       (1) pythonic -- list of Map instances where there is an entry in the Map
+               per column (key=column name, value=column value for that row)
+       (2) chapeltastic -- associative array with the key being the column name
+               and the value being an array of strings
+
+   You can compile and run this example by on the command line doing a
+       source util/quickstart/setchplenv.bash
+   in the chapel home directory, and then compiling and running this file as follows:
+       chpl --fast readcsv.chpl
```

Comment:
If this is just to demonstrate Chapel's capabilities, I don't think we should recommend `--fast` from the beginning. You lose a lot of debugging capabilities if you use `--fast`. An option would be adding a sentence, suggesting that `--fast` flag could be used for better performance at the cost of debuggability.

Reply:
This particular code is just way to slow otherwise.  4 minutes vs. less than a second.